testing/btest/Baseline.zam/broker.store.create-failure/zeek.err

```diff
@@ -2,8 +2,8 @@
 error in <...>/store.zeek, line 780: Failed to attach master store backend_failure: (Broker::__create_master(../fail, Broker::SQLITE, Broker::options.2))
 error in <...>/store.zeek, line 780: Could not create Broker master store '../fail' (Broker::__create_master(../fail, Broker::SQLITE, Broker::options.2))
 error in <...>/create-failure.zeek, line 49: invalid Broker store handle (Broker::keys(s) and broker::store::{})
-error in <...>/store.zeek, line 794: invalid Broker store handle (Broker::__close(Broker::h) and broker::store::{})
-error in <...>/store.zeek, line 794: invalid Broker store handle (Broker::__close(Broker::h) and broker::store::{})
+error in <...>/store.zeek, line 794: invalid Broker store handle (Broker::__close(Broker::h.12) and broker::store::{})
+error in <...>/store.zeek, line 794: invalid Broker store handle (Broker::__close(Broker::h.21) and broker::store::{})
```

Comment:
Total nit, what does `.12` and `.21` actually say? Are they useful enough to be user-facing?

Reply:
Not useful enough to be user-facing - they are variable renamings introduced by the inliner to avoid collisions. The actual number reflects where in the inlining process the variable was encountered.